rct229/ruletest_engine/ruletest_jsons/system_types/System_10_Warm_Air_Furnace_Elec.json

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "ASHRAE229 1",
+    "ruleset_model_instances": [
+        {
+            "id": "RMD 1",
+            "buildings": [
+                {
+                    "id": "Building 1",
+                    "building_open_schedule": "Required Building Schedule 1",
+                    "building_segments": [
+                        {
+                            "id": "Building Segment 1",
+                            "zones": [
+                                {
+                                    "id": "Thermal Zone 1",
+                                    "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                    "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                    "terminals": [
+                                        {
+                                            "id": "Air Terminal",
+                                            "is_supply_ducted": true,
+                                            "type": "CONSTANT_AIR_VOLUME",
+                                            "served_by_heating_ventilation_air_conditioning_system": "System 10",
+                                            "fan": {
+                                                "id": "Terminal Fan 1"
+                                            }
+                                        }
+                                    ]
+                                }
+                            ],
+                            "heating_ventilation_air_conditioning_systems": [
+                                {
+                                    "id": "System 10",
+                                    "heating_system": {
+                                        "id": "Furnace Coil 1",
+                                        "heating_system_type": "FURNACE",
```

Comment:
I believe ```heating_system_type``` shouldn't exist because of [this line](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/is_baseline_system_10.md#:~:text=hvac_b.heating_system.heating_system_type%20%3D%3D%20%22NONE%22%3A) in the RDS. Can you double-check this?